src/templates/compliance-buttons.js

```diff
@@ -0,0 +1,130 @@
+              </style>
+              <table id="buttons-1-0">
+                  <thead>
+                      <tr><th>Size</th><th>Light</th><th>Dark</th></tr>
+                  </thead>
+                  <tbody>
+                      <tr>
+                          <th>Small</th>
+                          <td><a href="/compliance-buttons/lottie-1.0-light-28.png">
+                              <img src="/compliance-buttons/lottie-1.0-light-28.png" alt="lottie-1.0-light-28.png" />
+                          </a></td>
+                          <td><a href="/compliance-buttons/lottie-1.0-dark-28.png">
+                              <img src="/compliance-buttons/lottie-1.0-dark-28.png" alt="lottie-1.0-dark-28.png" />
+                          </a></td>
+                      </tr>
+                      <tr>
+                          <th>Medium</th>
+                          <td><a href="/compliance-buttons/lottie-1.0-light-56.png">
+                              <img src="/compliance-buttons/lottie-1.0-light-56.png" alt="lottie-1.0-light-56.png" />
+                          </a></td>
+                          <td><a href="/compliance-buttons/lottie-1.0-dark-56.png">
+                              <img src="/compliance-buttons/lottie-1.0-dark-56.png" alt="lottie-1.0-dark-56.png" />
+                          </a></td>
+                      </tr>
+                      <tr>
+                          <th>Large</th>
+                          <td><a href="/compliance-buttons/lottie-1.0-light-112.png">
+                              <img src="/compliance-buttons/lottie-1.0-light-112.png" alt="lottie-1.0-light-112.png" />
+                          </a></td>
+                          <td><a href="/compliance-buttons/lottie-1.0-dark-112.png">
+                              <img src="/compliance-buttons/lottie-1.0-dark-112.png" alt="lottie-1.0-dark-112.png" />
+                          </a></td>
+                      </tr>
+                      <tr className="iframe-row">
+                          <th>Scalable</th>
+                          <td><a href="/compliance-buttons/lottie-1.0-light.svg">
+                              <iframe src="/compliance-buttons/lottie-1.0-light.svg" title="lottie-1.0-light.svg"></iframe>
+                          </a></td>
+                          <td><a href="/compliance-buttons/lottie-1.0-dark.svg">
+                              <iframe src="/compliance-buttons/lottie-1.0-dark.svg" title="lottie-1.0-dark.svg"></iframe>
+                          </a></td>
+                      </tr>
+                  </tbody>
+              </table>
+            </section>
+          </Col>
+        </Row>
+      </Container>
+    </Layout>
+  )
+}
+
+export default ComplianceButtons
+
+/**
+ * Head export to define metadata for the page
+ *
+ * See: https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-head/
+ */
+export const Head = () => (
+  <Seo
+    title="Compliance Buttons"
```

Comment:
Wondering if we should name this something more generic like "Public Assets" in case there are other things we want to make available to the community.

Reply:
Probably that's better kept on a different page. Once we get more versions out the buttons page will have enough stuff already :P